net/apinger/files/apinger.rpc

```diff
@@ -38,7 +38,7 @@ apinger_status() {
 		if [ -f "$status_file" ]; then
 			_IFS="$IFS"
 			IFS="|"
-			while read -r address srcip target received sent timestamp latency loss alarm; do
+			while read -r address srcip target sent received timestamp latency loss alarm; do
```

Comment:
Is this change needed

Reply:
According to the source, the status-format is written like this:

`fprintf(f,"%s|%s|%i|%i|%u",t->name, t->config->srcip, t->description, t->last_sent+1, t->received, t->last_received_tv.tv_sec);` - sent and then received

Without the change Sent and Received are shown incorrectly in luci. I noticed this when I only had sent packages without any received ones...